File: app/src/main/java/com/example/apptfg/Acerca_de.java
package com.example.apptfg;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class Acerca_de extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_acerca_de);
    }

    //Función para volver a la ventana anterior
    public void anteriorVentana(View view){
        Intent anterior = new Intent(this, Notificaciones.class);
        startActivity(anterior);
    }
}

File: app/src/main/java/com/example/apptfg/AcercaDe.kt
package com.example.apptfg

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

class AcercaDe : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_acerca_de)
    }

    //Función para volver a la ventana anterior
    fun anteriorVentana(view: View?) {
        val anterior = Intent(this, Notificaciones::class.java)
        startActivity(anterior)
    }
}